pkg/kms/localkms/pubkey_export_import_test.go

```diff
@@ -78,7 +96,7 @@ func TestPubKeyExportAndRead(t *testing.T) {
 				require.NotEmpty(t, verifier)
 
 				err = verifier.Verify(s, msg)
-				require.NotEmpty(t, err)
+				require.NoError(t, err)
```

Comment:
this fixes the current verification test... it was wrongly checking if err is not empty, instead of checking if there's no error